src/maxminddb.c

```diff
@@ -318,7 +322,7 @@ LOCAL int map_file(MMDB_s *const mmdb)
     if (INVALID_HANDLE_VALUE != fd) {
         CloseHandle(fd);
     }
-    if (INVALID_HANDLE_VALUE != mmh) {
+    if (INVALID_HANDLE_VALUE != mmh && NULL != mmh) {
```

Comment:
Does `INVALID_HANDLE_VALUE` ever return `INVALID_HANDLE_VALUE`? If not, we could just initialize `mmh` to `NULL` above and get rid of the `INVALID_HANDLE_VALUE` check here.